.travis.yml

```diff
@@ -59,6 +59,9 @@ jobs:
           arch: s390x
           compiler: gcc
           env: CONFIG_OPTS="--strict-warnings"
+        - os: linux-ppc64le
```

Comment:
I'm wondering why?

Reply:
It's quite possible that this platform demonstrated the issue at the time.
Since we don't do Travis any more, that's a bit hard to test for the moment...  I'm not sure if it's even possible to specify specific hardware with GitHub Actions